traffic_ops/install/bin/postinstall.py

```diff
@@ -0,0 +1,1344 @@
+#!/usr/bin/env -S sh -c 'PATH="${PATH}:/usr/libexec" exec "$(type -p python platform-python | head -n1)" "$0" "$@"'
```

Comment:
The Python postinstall script does not support Python 2.7, which is fine, but because it doesn't, it should search the patch for a `python3` explicitly, not `python`, else it would not work on CentOS 7.

Reply:
Done